chemicaltoolbox/openbabel/ob_remIons.xml

```diff
@@ -12,10 +12,12 @@
       -i '${infile}'
       -iformat '${infile.ext}'
       -o '${outfile}'
+      $index
 ]]>
     </command>
     <inputs>
         <expand macro="infile_all_types"/>
+        <param name="index" type="boolean" checked="false" truevalue="-idx" falsevalue="" label="Should output be indexed tab-separated table?" help="This option will work only when input format is InChI or SMILES, otherwise will be ignored!" />
```

Comment:
```suggestion
        <param name="index" type="boolean" checked="false" truevalue="-idx" falsevalue="" label="Output indexed tab-separated table?" help="Add an index column to the output file. This option will work only when input format is InChI or SMILES, otherwise it will be ignored." />
```

Reply:
Thanks, I will modify it ;)